src/core/catalog.js

```diff
@@ -638,10 +639,39 @@ class Catalog {
       return { name: stringToPDFString(nestedName), order: nestedOrder };
     }
 
+    function parseRBGroups(rbgrps) {
+      if (!Array.isArray(rbgrps)) {
+        return null;
+      }
+
+      // iterate over RB groups (literal arrays or refs thereof)
```

Comment:
*This comment applies in multiple other spots throughout the patch as well.*

To improve readability please remember that comments should be complete sentences, i.e. beginning with an upper-case letter and ending with a period.

```suggestion
      // Iterate over RB groups (literal arrays or refs thereof).
```